src/textual/widget.py

```diff
@@ -978,34 +977,36 @@ def _refresh_scrollbars(self) -> None:
         styles = self.styles
         overflow_x = styles.overflow_x
         overflow_y = styles.overflow_y
-        width, height = self.container_size
+        width, height = self._container_size
 
-        show_horizontal = self.show_horizontal_scrollbar
+        show_horizontal = False
```

Comment:
Can you help me understand why the default behaviour is to turn off `show_horizontal` instead of leaving the previous value?

Reply:
The previous value shouldn't come in to consideration.  It should make the same calculation, no matter the current state of the scrollbars.